pipeline/Dockerfile

```diff
@@ -6,8 +6,6 @@ WORKDIR /app
 RUN apt update && \
     apt install -y cmake
 
-RUN mkdir ~/.ssh
-RUN ssh-keyscan -H github.com >> ~/.ssh/known_hosts
```

Comment:
These ssh commands were mistakenly left in the Dockerfile from another PR when we tried to perform `git submodule update` for pdfalto through SSH. We switched to https as on line 13/11 as it was easier in CI.